.circleci/config.yml

```diff
@@ -154,7 +154,13 @@ run_e2e_tests: &run_e2e_tests
       DEV_TAG=$(latestReleaseTag)
       IMG_MODIFIER=""
     fi
-    ./script/e2e-test.sh $DEV_TAG $IMG_MODIFIER
+    if ./script/e2e-test.sh $DEV_TAG $IMG_MODIFIER; then
```

Comment:
I had to run the `e2e` tests as a conditional to be able to gather the screenshots afterwards (and not exiting directly)